### ISSUE TITLE
Errors when encountering FFI Call

### ISSUE THREAD

**Issue by tom95** [LinqLover/SimulationStudio#31]
On a recent squeak trunk image, I get an error in `SandboxContext>>doNamedPrimitiveIn:for:withArgs:` when a method with an FFI Call is simulated. Apparently the literal it contains does not conform to the SandboxContext's expectations (it is an ExternalLibraryFunction).

**Comment by LinqLover** [comment]
Thank you for the bug report! If you could provide an example literal here, I should be able to fix it pretty fast. Otherwise, I will take install FFI & take a look at it myself. :-)

**Comment by tom95** [comment]
"Literal" is maybe a misnomer here, it's an instance of an `ExternalLibraryFunction`. So I think for correct handling looking into it in more detail is probably a good idea. Here's an example to reproduce:

```smalltalk
MyObj>>testFFI

	<apicall: void here() module: 'a'>


Sandbox new evaluate: [MyMorph basicNew testFFI ]
```

**Comment by LinqLover** [comment]
Alright, should be fixed via #32. Please note that in the sandbox, FFI calls are forbidden at the moment because they could have unintended side effects. (However, it would be possible to allow-list certain FFI functions that are known to work side-effect free.) In any way, there won't be an exception on the simulator stack any longer but on the simulated process/stack a `SimulationOperationForbidden` error is raised. For type guessing in Sandblocks, this means that currently, no guessed type will be available for expressions that include an FFI call. Please report back if you see anything else that needs to be changed. :-)

**Comment by tom95** [comment]
Works perfectly, thank you!

Yeah, FFI is a difficult topic in this regard, as it is prone to things like segfaults even if the operation itself *should* be side-effect free. Plus, step/time limits etc can obviously become quite inaccurate for long running external calls (which I suppose counts for primitives as well though). For my use cases, simply aborting is fine :)
